### ISSUE TITLE
Regression for 404

### ISSUE THREAD

**Issue by stevepiercy** [plone/documentation#1673]
### Description

See https://github.com/plone/documentation/issues/1399

We want a default 404 to look like this:

https://6.docs.plone.org/404.html

Not like this:

https://6.docs.plone.org/whoopsie

Something regressed since this was deployed.

**Comment by polyester** [comment]
Indeed a whoopsie. Custom 404 isn't properly set up in Ansible. Manually fixed for now, leaving issue open until I can fix that Ansible template.

**Comment by stevepiercy** [comment]
Do you know when it regressed? I only noticed it because the 404 report in Matomo collected no data. I might have not set up the file correctly.

@fredvd does the 404 view require its own JavaScript snippet for Matomo to count it?

- https://matomo.org/blog/2019/07/how-to-analyse-404-pages/
- https://matomo.org/faq/how-to/faq_60/?

**Comment by polyester** [comment]
Yes, happened when I ran ansible to put in a new redirect. June 9th, 2024. 

Mea Culpa `<applies ash generously on head/>`

**Comment by polyester** [comment]
@stevepiercy as the server for 'static sites' holds quite a lot of those, it will require some planning to make sure that "custom 404 pages" will work without disrupting other sites, as the current setup pulls in a snippet for *all* sites on error pages. That needs re-working and a bit of planning. Will get on it.

**Comment by stevepiercy** [comment]
> Yes, happened when I ran ansible to put in a new redirect. June 9th, 2024.

That's much more recent than when we thought we configured Matomo and the documentation 404 template correctly. In fact, no 404 data has been collected ever. I think I have a fix for it, using a custom JavaScript snippet, but need to check with @fredvd.
